samples/asgardeo-express-app/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "asgardeo-express-sample",
```

```suggestion
  "name": "asgardeo-express-app",
```
Check other sample apps to check if this is correct.

---

Should be `@asgardeo/express-app`